src/backend/bs_admin

```diff
@@ -70,6 +70,25 @@ my $sourcedb = "$BSConfig::bsdir/db/source";
 my $rundir  = "$BSConfig::bsdir/run";
 my $dodsdir = "$BSConfig::bsdir/dods";
 
+my $sc_help = <<SC_HELP;
+Service Containers
+==================
+
+  --sc <podman_command> [sc_options] [podman_options]
```

Comment:
would be nice to have short and a long speaking option name to be consistent with the other switches.